test-environment.yml

```
@@ -1,14 +1,14 @@
 name: bqplot-test
 channels:
     - conda-forge
-    - defaults
 dependencies:
     - python
     - ipywidgets >=5.2.2
     - traitlets >=4.3.0
     - traittypes >=0.0.6
     - numpy >=1.10.4
     - pandas
+    - jupyterlab=3.0.0  # to build the lab federated bundle
```

Should we relax the pin?

```suggestion
    - jupyterlab=3  # to build the lab federated bundle
```